install-components.md

```diff
@@ -1075,7 +1075,7 @@ To install Tanzu Build Service using the Tanzu CLI:
     tanzunet_username: TANZUNET-USERNAME
     tanzunet_password: TANZUNET-PASSWORD
     descriptor_name: DESCRIPTOR-NAME
-    enable_automatic_dependency_updates: true
+    enable_automatic_dependency_updates: "true-or-false" # Optional 
```

Comment:
The quotes here kind of look like the value "true-or-false" is expected. I am not sure if this is better but, maybe?
```suggestion
    enable_automatic_dependency_updates:  # Optionally either "true" or "false"
```

Reply:
agreed except that true/false shouldn't be quoted I think that could be confusing